spec/controllers/account/presentations_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe Account::PresentationsController, type: :controller do
+  render_views
+
+  describe 'GET #index' do
+    context 'with signed in user' do
+      let!(:user) { create(:user) }
+
+      before(:each) do
+        sign_in user
+        get :index
+      end
+
+      subject { response }
+
+      it { is_expected.to have_http_status(200) }
+      it { is_expected.to render_template('index') }
```

Comment:
check in "index" test that title of user presentation is rendered on INDEX page in body (prepare test data)